chainer/training/extensions/multistep.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import division
+
+from chainer.training import extension
+
+
+class Multistep(extension.Extension):
```

Comment:
* Please name it `*Shift`.
* Please add docstring for this class. c.f. https://github.com/chainer/chainer/pull/4894/files
* Please add it to the docs. (`docs/reference/training.rst`)
https://docs.chainer.org/en/latest/reference/training.html#optimizer-behavior-control